src/classlibnative/inc/nlsinfo.h

```diff
@@ -145,53 +86,7 @@ class COMNlsInfo {
 
     static void QCALLTYPE nativeNormalizationInitNormalization(int NormForm, BYTE* pTableData);
```

Comment:
Why do we need these normalization fcalls? I though that you have ported over the implementation that just PInvokes directly.